BitwardenShared/UI/Vault/Vault/VaultItemSelection/VaultItemSelectionView.swift

```diff
@@ -104,25 +111,40 @@ private struct VaultItemSelectionSearchableView: View {
                 get: \.toast,
                 send: VaultItemSelectionAction.toastShown
             ))
+            .background(Color(asset: Asset.Colors.backgroundSecondary).ignoresSafeArea())
     }
 
     // MARK: Private Views
 
     /// The content displayed in the view.
     @ViewBuilder
     private func contentView() -> some View {
-        VStack(spacing: 0) {
-            if isSearching {
-                searchContentView()
-            } else {
-                if store.state.vaultListSections.isEmpty {
-                    // TODO: BIT-2350 Add empty view
-                } else {
-                    matchingItemsView()
+        if isSearching {
+            searchContentView()
```

Comment:
🤔 According to **Codecov** a test is missing for when `isSearching` is `true`.

Reply:
Alright, added search snapshots.